### PR TITLE
[sc-4033] dispatch orientation change event

### PR DESCRIPTION
**Pull request by StephanieZuniga**

### CODE REVIEW

File: MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift [hunk 1]
```diff
@@ -773,6 +773,35 @@ extension MUXSDKPlayerBinding {
...
+        self.updateVideoData(player: player)
+        
+        var orientationData: MUXSDKViewDeviceOrientationData? = nil
+        
+        switch orientation {
+        case.landscape:
+            orientationData = MUXSDKViewDeviceOrientationData(z: 0.0)
+        case .portrait:
+            orientationData = MUXSDKViewDeviceOrientationData(z: 90.0)
+        case .unknown:
```
Comment (nidhik) [inline MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift:792]: let's log a warning here also?

File: MUXSDKKaltura/MUXSDKKaltura/Models/MUXSDKViewOrientation.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+//  Mux-Stats-Kaltura
+//
+//  Created by Stephanie Zuñiga on 4/11/21.
+//  Copyright © 2021 Mux, Inc. All rights reserved.
+//
+
+import Foundation
+
+public enum MUXSDKViewOrientation {
+    case unknown
```
Comment (nidhik) [inline MUXSDKKaltura/MUXSDKKaltura/Models/MUXSDKViewOrientation.swift:12]: wondering if we need the `unknown` case - this method is designed to be called by the app developer and they should know if it is portrait or landscape?

File: MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift [hunk 3]
```diff
@@ -773,6 +773,35 @@ extension MUXSDKPlayerBinding {
         self.dispatcher.dispatchEvent(event, forPlayer: self.name)
         self.state = .viewEnd
     }
+    
+    func dispatchOrientationChange(orientation: MUXSDKViewOrientation) {
```
Comment (nidhik) [inline MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift:777]: can we add a unit test for this?

File: MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift (hunk 1 above)
Comment (StephanieZuniga) [reply MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift:792]: I think you're right we should not support unknown. Im removing it.

File: MUXSDKKaltura/MUXSDKKaltura/Models/MUXSDKViewOrientation.swift (hunk 2 above)
Comment (StephanieZuniga) [reply MUXSDKKaltura/MUXSDKKaltura/Models/MUXSDKViewOrientation.swift:12]: I dont think we need it I am removing it.

File: MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift (hunk 3 above)
Comment (StephanieZuniga) [reply MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift:777]: Sure, done.

**Pull request merged**